docs/release/3.3/release-3-3-30-3.md

```diff
@@ -0,0 +1,82 @@
+OSG Software Stack -- Data Release -- 3.3.30-3
+==============================================
+
+**Release Date**: 2017-11-29
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.88](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    - updated UKeScience 2B ICA based on a SHA-2 family digest (UK)
+    - added new PKIUNAMgrid (2017) trust anchor for roll-over (MX)
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.3.30-3%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found [here](../../).
+
+Updating to the new release
+---------------------------
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
+
+### Update Software
+
+Once the repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note
+    Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
+
+!!! note
+    Watch the yum update carefully for any messages about a `.rpmnew` file being created. That means that a configuration file had been editted, and a new default version was to be installed. In that case, RPM does not overwrite the editted configuration file but instead installs the new version with a `.rpmnew` extension. You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version into place (that is, without the `.rpmnew` extension). Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
```

Comment:
I just discussed this with @matyasselmeci and we don't need to stress `/etc/lcmaps.db` anymore. Same goes for 3.3